ui/src/app/utils/runtime-utils.spec.tsx

```diff
@@ -23,34 +45,43 @@ import {
 import defaultServerConfig from 'testing/default-server-config';
 import { stubDisk } from 'testing/stubs/disks-api-stub';
 import {
+  defaultDataprocConfig,
   defaultDataProcRuntime,
   defaultGceRuntime,
   defaultGceRuntimeWithPd,
   defaultRuntime,
   RuntimeApiStub,
 } from 'testing/stubs/runtime-api-stub';
 
-const WORKSPACE_NS = 'test';
+import {
+  allMachineTypes,
+  ComputeType,
+  DEFAULT_DISK_SIZE,
+  DEFAULT_MACHINE_TYPE,
+} from './machines';
+import { runtimePresets } from './runtime-presets';
 
-const Runtime = ({ id }) => {
```

Comment:
this was causing headaches after I needed to import our standard 'Runtime' so I renamed it